src/content/en/fundamentals/accessibility/accessible-styles.md

```diff
@@ -0,0 +1,364 @@
+style that is only applied if the element is focused via the keyboard, quite a
+handy feature. While this pseudo-class is currently only supported in Firefox,
+[there is currently work going on to turn it into a
+standard](https://github.com/wicg/modality).
+
+There is also <a
+href="http://radar.oreilly.com/2015/08/proposing-css-input-modailty.html"
+target="_blank">this great article by Alice Boxhall and Brian Kardell</a> that
+explores the topic of modality and contains prototype code for differentiating
+between mouse and keyboard input. You can use their solution today, and then
+include the focus ring pseudo-class later when it has more widespread support.
+
+## Styling states with ARIA
+
+When you build components, it's common practice to reflect their state, and thus
+their appearance, using CSS classes controlled with JavaScript.
+
+For example, consider a toggle button that goes into a "pressed" visual state
+when clicked and retains that state until it is clicked again. To style the
+state, your JavaScript might add a `pressed` class to the button. And, because
+you want good semantics on all your controls, you would also set the
+`aria-pressed` state for the button to `true`.
+
+A useful technique to employ here is to remove the class altogether, and just
+use the ARIA attributes to style the element. Now you can update the CSS
+selector for the pressed state of the button from this
+
+
+    .toggle.pressed { ... }
+    
+
+to this.
+
+
+    .toggle[aria-pressed="true"] { ... }
+    
+
+This creates both a logical and a semantic relationship between the ARIA state
+and the element's appearance, and cuts down on extra code as well.
+
+## Multi-device responsive design
+
+We know that it's a good idea to design responsively to provide the best
+multi-device experience, but responsive design also yields a win for
+accessibility.
+
+Consider a site like <a href="https://www.udacity.com/"
+target="_blank">Udacity</a>. A low-vision user who has difficulty reading small
+print might zoom in the page, perhaps as much as 400%, whereupon they are now
+essentially viewing the mobile version of the site on their desktop.
+
+Because the site is designed responsively, the UI will rearrange itself for the
+"smaller viewport" (actually for the larger page), which is great for desktop
+users who require screen magnification and for mobile screen reader users as
+well. It's a win-win. In fact, just by designing responsively, we're meeting <a
+href="http://webaim.org/standards/wcag/checklist#sc1.4.4" target="_blank">rule
+1.4.4 of the WebAIM checklist</a>, which states that a page "...should be
+readable and functional when the text size is doubled."
+
+Going over all of responsive design is outside the scope of this guide, but
+here are a few important takeaways that will benefit your responsive experience
+and give your users better access to your content.
+
+ - First, make sure you always use the proper `viewport` meta tag.
+
+   ```
```

Comment:
Can't seem to get this to work. Suggested I leave it as is and Pete will take a look